<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Schet</class>
 <widget class="QDialog" name="Schet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Счёт клиента</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QPushButton" name="btnConnect">
     <property name="text">
      <string>Подключение</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="leSearch"/>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="btnSelectAll">
     <property name="text">
      <string>ВЫбрать Все</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbIDs">
        <property name="text">
         <string>Номер счета</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLineEdit" name="leIDSchet"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbCur">
        <property name="text">
         <string>Валюта</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QLineEdit" name="leCur"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbBalance">
        <property name="text">
         <string>Баланс</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QLineEdit" name="leBalance"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbData">
        <property name="text">
         <string>Дата</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" rowspan="2" colspan="2">
       <widget class="QLineEdit" name="leData"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="lbTextData">
        <property name="text">
         <string>открытия счёта</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QPushButton" name="btnEdit">
        <property name="text">
         <string>Изменить</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="btnDel">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="QTableWidget" name="twSec"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btnSearch">
     <property name="text">
      <string>Поиск</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnExit">
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2" colspan="4">
    <widget class="QTableWidget" name="twSchet"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
